commands/root.go

```diff
@@ -19,12 +19,13 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/spf13/cobra"
+
 	clusterDir "github.com/hazelcast/hazelcast-commandline-client/commands/cluster"
 	mapDir "github.com/hazelcast/hazelcast-commandline-client/commands/types/map"
-	"github.com/spf13/cobra"
 )
 
-const DefaultConfigFile = ".hzc.yaml"
+const DefaultConfigFilePath = ".local/share/hz-cli/.hzc.yaml"
```

Comment:
Let's remove this constant and have a `homeDir(path)` function which would return something like:
```
filepath.Join(os.UserHomeDir(), ".local/share/hz-cli". path)
```

Reply:
It seems we do not use config path for anything else. 

`func DefaultConfigFile() string {
	homeDirectoryPath, _ := os.UserHomeDir()
	return filepath.Join(homeDirectoryPath, DefaultConfigPathFromHomeDir, DefaultConfigFilename)
}`

What do you think for this instead of "homedir" func?

Reply:
If we have the `homeDir` function, `DefaultConfigPath` function could be written as:
```
func DefaultConfigPath() string {
    return homeDir("config.yaml")
}
```
But if you think we won't have anything other than the configuration file in the home directory, it is OK to only have the `DefaultConfigPath` function. We may later add `homeDir` if necessary.

Reply:
Sounds great, thanks